pytest_reportportal/service.py

```diff
@@ -626,16 +628,28 @@ def get_marker_value(item, keyword):
             return "{}:{}".format(keyword, marker.args[0]) \
                 if marker and marker.args else keyword
 
+        # Get launch and test attributes and converting them to key:value form
+        def get_raw_attr(get_marker):
+            raw_attr = [
+                get_marker_value(item, k)
+                for k in item.keywords
+                if get_marker(k) is not None
+                and k not in self.ignored_attributes
+            ]
+
+            raw_attr.extend([
+                tag
+                for tag in item.session.config.getini('rp_tests_attributes')
+            ])
+            return raw_attr
+
         try:
             get_marker = getattr(item, "get_closest_marker")
         except AttributeError:
             get_marker = getattr(item, "get_marker")
-        attributes = [{"value": get_marker_value(item, k)}
```

Comment:
Sure, you're right. Only a small clarification: getini() already returns a list so there is no need in using brackets

Reply:
Correct. You have resolved this issue and haven't changed anything...

Reply:
I changed this line. `raw_attr.extend(item.session.config.getini('rp_tests_attributes'))`
Or you want me to change the look of list comprehension as well?

Reply:
I have updated it as you suggested

Reply:
No. The suggested code block is the only change that needs to be done in that method. There is no sense in moving a part of functionality into the get_raw_attr() because this function is used only once.

Reply:
Understood. Now it looks as you explained